js/grocy.js

```diff
@@ -150,21 +152,21 @@ class GrocyClass
 
 	translate(text, ...placeholderValues)
 	{
-		if (this.Mode === "dev")
+		/*if (this.Mode === "dev")
```

Comment:
intentional?

Reply:
Semi. I was toying around with some Ideas how to extract translations statically using https://github.com/sinedied/gettext-extract, but reverted the changes. This seems to have fallen through.